func-complete-check/check-validator/v1.ts

```diff
@@ -0,0 +1,81 @@
+import { Context } from '@azure/functions'
+import { ValidateCheckMessageV1, ReceivedCheck, MarkCheckMessageV1 } from '../typings/message-schemas'
+import Moment from 'moment'
```

Comment:
Why uppercase on moment?

Reply:
that is the type that is exported from the TS definition file